naresh/src/main/java/com/lftechnology/phpjava/menu/ProgramMenu.java

```diff
@@ -0,0 +1,71 @@
+package com.lftechnology.phpjava.menu;
+
+import com.lftechnology.phpjava.assignmentthree.EvenFibonacci;
+import com.lftechnology.phpjava.assignmentthree.LargestPalindrome;
+import com.lftechnology.phpjava.assignmenttwo.MutiplierChecker;
+import com.lftechnology.phpjava.assignmenttwo.SalutationSplitter;
+
+import java.util.Scanner;
+
+/**
+ * Created by naresh on 5/16/16.
+ */
+public class ProgramMenu {
+    public static String[] menu = {
+            "1. Split Salutation",
+            "2. Check Multiple",
+            "3. Find Largest Palindrome",
+            "4. Even Fibonacci",
+            "0. To Exit",
+    };
+
+    public static void main(String[] args) {
+        showMenu();
+    }
+
+    public static void showMenu() {
+        System.out.println("Please enter the option number");
+        Scanner scan = new Scanner(System.in);
+        for (String option : menu) {
+            System.out.println(option);
+        }
+        int index = scan.nextInt();
+        run(index);
+    }
+
+    public static void run(int index) {
+        switch (index) {
+            case 1:
+                SalutationSplitter salutationSplitter = new SalutationSplitter();
+                salutationSplitter.scanner();
+                break;
+            case 2:
+                MutiplierChecker mutiplierChecker = new MutiplierChecker();
+                mutiplierChecker.scanner();
+                break;
+            case 3:
+                LargestPalindrome largestPalindrome = new LargestPalindrome();
+                largestPalindrome.scanner();
+                break;
+            case 4:
+                EvenFibonacci evenFibonacci = new EvenFibonacci();
+                evenFibonacci.scanner();
+                break;
+            case 0:
+                System.out.println("Tata Horn OK please. :)");
+                System.exit(0);
+            default:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 11 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1151)